src/css/doc.css

```diff
@@ -499,6 +499,8 @@ ul ul ul {
 
 .doc .abstract blockquote {
   margin: 0;
+  padding-left: 0.5em;
```

Comment:
Thoughts on tweaking this to `1.5em` so we get something more like: 

![image](https://github.com/couchbase/docs-ui/assets/110928505/62050589-ae77-4985-9ca8-83b742f31b56)

Reply:
That looks fine to me 👍